microcosm_pubsub/tests/conventions/test_messages.py

```diff
@@ -102,17 +189,17 @@ def test_decode_identity_message_schema():
     Message decoding should process standard IdentityMessage fields.
 
     """
-    schema = IdentityMessageSchema(make_media_type("Foo"))
-    codec = PubSubMessageCodec(schema)
+    identity_schema = IdentityMessageSchema(make_media_type("Foo"))
+    codec = PubSubMessageCodec(identity_schema)
     message = dumps({
-        "mediaType": "application/vnd.globality.pubsub.foo",
+        "mediaType": "application/vnd.globality.pubsub._.created.foo",
```

Comment:
This previously made no sense; if we were decoding a message that was associated with a specific message, we should be validating that media type.

Come to think of it, we should probably do that validation during load.

Reply:
Do you mean "a message that was associated with a specific media type"?

Reply:
True, I'm probably misunderstanding some usage here. When a message is deserialized, we use this key to determine which schema to decode it with; it doesn't make sense, then, that the test is using a mismatched media type.
So yeah, validation not needed; we get that via the registry.